docs/docs/installation.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Installation
+title: Download
```

Comment:
Hopefully more clear so the sidebar workflow is:

1. What is pomerium
2. How do I get it